test/TestConfig/makefile

```diff
@@ -69,4 +69,4 @@ getdependency:
 # compile all tests under $(TEST_ROOT)
 #######################################
 compile: getdependency
-	ant -f scripts$(D)build_test.xml -DTEST_ROOT=$(TEST_ROOT) -DBUILD_ROOT=$(BUILD_ROOT) -DJAVA_BIN=$(JAVA_BIN) -DJAVA_VERSION=$(JAVA_VERSION) -DJCL_VERSION=$(JCL_VERSION) -DBUILD_LIST=${BUILD_LIST} -DRESOURCES_DIR=${RESOURCES_DIR} -DJCK_ROOT=$(JCK_ROOT)
+	ant -f scripts$(D)build_test.xml -DTEST_ROOT=$(TEST_ROOT) -DBUILD_ROOT=$(BUILD_ROOT) -DJAVA_BIN=$(JAVA_BIN) -DJAVA_VERSION=$(JAVA_VERSION) -DJCL_VERSION=$(JCL_VERSION) -DBUILD_LIST=${BUILD_LIST} -DRESOURCES_DIR=${RESOURCES_DIR} -DJAVA_IMPL=${JAVA_IMPL} -DJCK_ROOT=$(JCK_ROOT)
```

Comment:
Do you mind of changing { } to ( ) for all passing values?

Reply:
Updated.